hatch/src/deps/mod.rs

```diff
@@ -59,25 +60,38 @@ pub fn clone_project_deps(path: &Path,
     let hatchfile = hatchfile_path(dir);
 
     if hatchfile.exists() {
-      return clone_nested_project_deps(&registry, &dir, &mut visited);
+      return clone_nested_project_deps(&registry, &dir, &mut errored, &mut visited);
     }
     Ok(true)
   })?;
 
+  if !errored.is_empty() {
+    errored.into_iter().for_each(|e| {
+      println!("ERROR: {}", e)
+    });
+  }
+
   Ok(())
 }
 
 fn clone_nested_project_deps(registry: &Path,
                              path: &Path,
-                             visited: &mut
-                             HashSet<String>) -> HatchResult<bool>
+                             errored: &mut Vec<HatchError>,
+                             visited: &mut HashSet<String>) -> HatchResult<bool>
 {
-  let current_project = task::read_project(path)?;
-  if !visited.contains(&current_project.name().to_owned()) {
-    current_project.deps().iter().for_each(|dep| {
-      clone_dep(&dep.url(), &registry.join(dep.name()).as_path());
-    });
-    let _ = visited.insert(current_project.name().to_owned());
-  }
-  Ok(true)
+    match task::read_project(path) {
+      Err(e) => {
+        errored.push(e);
+        return Ok(true);
```

Comment:
We explicitly return `Ok(true)`, but it looks like we always return `Ok(true)` anyways. Is this explicit return necessary?